coroutine-utils/build.gradle.kts

```
@@ -10,4 +10,5 @@ plugins {
 dependencies {
   implementation(libs.kotlin.coroutines.core)
   implementation(libs.dagger.hilt.core)
+  implementation(libs.thirdparty.kotlinResult)
```

```suggestion
  api(libs.thirdparty.kotlinResult)
```

Your public API includes types from the library so this is supposed to be an api dependency

---

Oh, missed that, my bad!